gensim/utils.py

```diff
@@ -1005,15 +1004,13 @@ def pyro_daemon(name, obj, random_suffix=False, ip=None, port=None, ns_conf={}):
 
 def has_pattern():
     """
-    Function to check if there is installed pattern library
+    Function which returns a flag indicating whether pattern is installed or not
     """
-    pattern = False
     try:
-        from pattern.en import parse
-        pattern = True
-    except ImportError:
-        warnings.warn("Pattern library is not installed, lemmatization won't be available.")
-    return pattern
+        import pattern
+        return True
+    except:
```

Comment:
This method is just for testing if pattern is present or not, later to be called in wikicorpus method. Will adding ImportError make any difference? 

Sorry I am not very good with exception handling. We hardly use them in our academic setting.